Digipost/POSDocumentsViewController.m

```diff
@@ -521,7 +521,7 @@ - (void)updateContentsFromServerUserInitiatedRequest:(NSNumber *)userDidInititat
         
         [self showTableViewBackgroundView:([self numberOfRows] == 0)];
         if ([userDidInititateRequest boolValue]){
-            [UIAlertController presentAlertControllerWithAPIError:error presentingViewController:self];
+            [UIAlertController presentAlertControllerWithAPIError:error presentingViewController:self didTapOkClosure:nil];
```

Comment:
I actually found a pretty insane bug in the swift compiler/translator to objective-C when trying this.
 Sadly it won't compile at present Xcode 6.3 with Swift 1.2 :sob: